cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/optimized/MetaDataApiModel.kt

```diff
@@ -20,7 +20,8 @@ internal data class MetaDataParamReq(
 @Serializable
 data class MetaDataResp(
     @SerialName("ccpa") val ccpa: Ccpa?,
-    @SerialName("gdpr") val gdpr: Gdpr?
+    @SerialName("gdpr") val gdpr: Gdpr?,
+    @SerialName("usnat") val usNat: USNat?
```

Comment:
@carmelo-iriti please, add `,` at the end of the line to prevent GH from highlighting it if we would to add something below `usnat`